verification/curator-service/ui/cypress/integration/components/BulkCaseForm.spec.ts

```diff
@@ -59,20 +59,36 @@ describe('Bulk upload form', function () {
         // Demographics
         cy.contains('42-43');
         cy.contains('Male');
+        cy.contains('Accountant');
+        cy.contains('Bangladeshi');
+        cy.contains('British, Indian');
 
         // Location
-        cy.contains('Admin3');
         cy.contains('London, Greater London, England, United Kingdom');
+        cy.contains('Admin3');
 
         // Events
-        // Confirmed case date
+        // Confirmation
         cy.contains('2020-6-23');
+        cy.contains('PCR test');
+        // Symptom onset
+        cy.contains('2020-6-19');
         // Hospital admission
         cy.contains('Yes');
+        cy.contains('2020-6-21');
+        // ICU admission
         cy.contains('2020-6-22');
         // Outcome
         cy.contains('Recovered');
         cy.contains('2020-6-24');
+
+        // Symptoms
+        cy.contains('Symptomatic');
+        cy.contains('cough, fever');
+
+        // Preexisting conditions
+        cy.contains('Yes');
```

Comment:
README says true/false -- do we remap to yes/no?

Reply:
We do -- this is a boolean in the data model, but the UI converts that to a `Yes` (or presumably `No`) here.